### PR TITLE
passkey fail message

### PR DESCRIPTION
**Pull request by stormcloud266**
fix this ticket
https://3.basecamp.com/5743564/buckets/36023883/todos/7589837995

### CODE REVIEW

**Comment by stormcloud266** [comment]
* **#578** <a href="https://app.graphite.dev/github/pr/0xsend/sendapp/578?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `dev`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @stormcloud266 and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: packages/app/features/auth/sign-in/sign-in-form.tsx [hunk 1]
```diff
@@ -15,6 +15,13 @@ import { useState } from 'react'
 import { useRouter } from 'solito/router'
 import { bytesToHex, hexToBytes } from 'viem'
 
+const formatErrorMessage = (error: Error) => {
```
Comment (0xBigBoss) [inline packages/app/features/auth/sign-in/sign-in-form.tsx:18]: this is good. potentially we may want to add it to our userop or passkey library so we don't have to copy-pasta this everywhere.

https://github.com/0xsend/sendapp/blob/4772ca506a4225a7263074768b3fb3a5e582730a/packages/app/utils/userop.ts#L133

**Review by 0xBigBoss — APPROVED** [review]

File: packages/app/features/auth/sign-in/sign-in-form.tsx (hunk 1 above)
Comment (stormcloud266) [reply packages/app/features/auth/sign-in/sign-in-form.tsx:18]: my next ticket is to update another error message. if i move that particular function to userop, does it make sense to use it in both places?

full

Comment (0xBigBoss) [reply packages/app/features/auth/sign-in/sign-in-form.tsx:18]: This is different, I believe this error comes back from the 4337 RPC bundler.

Comment (0xBigBoss) [reply packages/app/features/auth/sign-in/sign-in-form.tsx:18]: Pretty sure that error gets thrown from here https://github.com/0xsend/sendapp/blob/4772ca506a4225a7263074768b3fb3a5e582730a/packages/app/utils/useUserOpTransferMutation.ts#L121-L123

Comment (0xBigBoss) [reply packages/app/features/auth/sign-in/sign-in-form.tsx:18]: Our userop logic may not be centralized though, I hope I centralized the logic to that method.

**Pull request merged**